### DISCUSSION TITLE
help on steps to get the examples to work ?

### DISCUSSION THREAD

**Discussion by rubricsinger** [perspective-dev/perspective#2024] [Q&A] [answered]
Hi, this will probably seem the most stupid question, but having followed the steps to install, which appears to have worked successfully.  I thought i would start by trying out the examples in the examples directory.     I was specifically keen to try out the python-tornado-streaming example.  the server.py files seems to work just fine, but the accompanying index.html file appears to reference js files that i just cannot find anywhere on my install, see below :-   

`<script src="http://localhost:8080/node_modules/perspective-viewer/dist/umd/perspective-viewer.js"></script>
        <script src="http://localhost:8080/node_modules/perspective-viewer-datagrid/dist/umd/perspective-viewer-datagrid.js"></script>
        <script src="http://localhost:8080/node_modules/perspective-viewer-d3fc/dist/umd/perspective-viewer-d3fc.js"></script>
        <script src="http://localhost:8080/node_modules/perspective/dist/umd/perspective.js"></script>
`

Is there an obvious build step that I have missed to generate the js files in some way?   I have seached extensively so far for any other questions like this and not seen them.    Many thanks.

**Comment by rubricsinger** [comment, marked as answer]
Duh !  ok so I resolved this by reading the instructions at the top of the javascript users guide :- [Here](https://perspective.finos.org/docs/js.html)

Basically I had not grasped that I needed to run the webpack install via yarn add.

**Reply by timkpaine** [reply]
You may also find this example helpful https://github.com/timkpaine/perspective-workspace-react-tornado

**Reply by rubricsinger** [reply]
Thanks loads I will definitely take a look.
